src/WebJobs.Script.Grpc/Server/IHttpProxyService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Yarp.ReverseProxy.Forwarder;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    public interface IHttpProxyService
+    {
+        public ValueTask<ForwarderError> Forward(ScriptInvocationContext context, Uri httpUri);
```

Comment:
nit: 
```suggestion
        public ValueTask<ForwarderError> ForwardAsync(ScriptInvocationContext context, Uri httpUri);
```

Also, should this take in a `CancellationToken`? Or is one available on the context object?

Reply:
@jviau Yes `CancellationToken` implementation is something on the to-do list across the entire flow between host and worker.